README.md

```diff
@@ -261,4 +266,153 @@ Once done, you need to start all components involved, these are:
+   18:33:16,282 |-INFO in ch.qos.logback.classic.joran.action.LoggerAction - Setting level of logger [org.modelix] to DEBUG
+   18:33:16,282 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [console] to Logger[org.modelix]
+   18:33:16,282 |-INFO in ch.qos.logback.classic.joran.action.RootLoggerAction - Setting level of ROOT logger to INFO
+   18:33:16,282 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [console] to Logger[ROOT]
+   18:33:16,282 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - End of configuration.
+   18:33:16,283 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@7ff95560 - Registering current configuration as safe fallback point
+   18:33:16.295 [main] INFO  org.modelix.model.server.Main - Max memory (bytes): 32178700288
+   18:33:16.295 [main] INFO  org.modelix.model.server.Main - Max memory (bytes): 32178700288
+   18:33:16.295 [main] INFO  org.modelix.model.server.Main - Server process started
+   18:33:16.295 [main] INFO  org.modelix.model.server.Main - Server process started
+   18:33:16.295 [main] INFO  org.modelix.model.server.Main - In memory: true
+   18:33:16.295 [main] INFO  org.modelix.model.server.Main - In memory: true
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Path to secret file: /secrets/modelsecret/modelsecret.txt
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Path to secret file: /secrets/modelsecret/modelsecret.txt
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Path to JDBC configuration file: null
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Path to JDBC configuration file: null
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Schema initialization: false
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Schema initialization: false
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Set values: []
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Set values: []
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Port: 28101
+   18:33:16.296 [main] INFO  org.modelix.model.server.Main - Port: 28101
+   Values loaded from /home/nkoester/git/modelix/modelix-sample/model-server/courses.modelserver.dump (73)
+   18:33:16.364 [main] INFO  ktor.application - Autoreload is disabled because the development mode is off.
+   18:33:16.428 [main] INFO  ktor.application - Application started in 0.088 seconds.
+   18:33:16.518 [DefaultDispatcher-worker-1] INFO  ktor.application - Responding at http://0.0.0.0:28101
+   <===========--> 85% EXECUTING [7s]
+   > :model-server:run
+
+   ```
+
+   </details>
+
+2. **API layer**: The `rest-api-model-server` provides an abstraction of the model from the previously started `model-server`, simply run (it will be a blocking call):
```

Comment:
Maybe some hint to use multiple terminals? ;-)

Reply:
"it will be a blocking call" = this will block your terminal? but I can be more explicit...